src/components/Counter.js

```diff
@@ -0,0 +1,39 @@
+import React, { createElement } from 'react';
+
+class Counter extends React.Component {
+    state = {count: 0}
```

Comment:
Missed requirement - The component should take an initial value in a property.

```suggestion
    constructor(props) {
        super(props);
        
        this.state = { count: this.props.initialValue };
    }
```
or 
```suggestion
       state = { count: this.props.initialValue };
```

Reply:
@MykhailoSheliahov fixed in module 2 and module 3